### PR TITLE
Test for recurringpaymentpush

### PR DESCRIPTION
**Pull request by osahyoun**

### CODE REVIEW

**Comment by osahyoun** [comment]
This is the new endpoint.

File: app/workers/queue_listener.rb [hunk 1]
```diff
@@ -78,6 +82,10 @@ def create_donation(params)
     client.create_donation(data)
   end
 
+  def create_payment(params)
+    client.create_recurring_payment(params[:params])
+  end
```
Comment (osahyoun) [inline app/workers/queue_listener.rb]: I've already added and merged this method to the AK gem.

File: spec/requests/actions_spec.rb [hunk 2]
```diff
@@ -8,6 +8,31 @@
...
+          post '/message', params
+        end
+      end
+
+      # Confirm transaction has been created here:
+      #   https://act.sumofus.org/rest/v1/donationaction/67503908/
+      #
+      it "is successful" do
+        expect(response.status).to eq(200)
+      end
```
Comment (osahyoun) [inline spec/requests/actions_spec.rb]: ActionKit doesn't return anything, so can only test status.

**Comment by NealJMD** [comment]
this looks good to me!

**Pull request merged**